library/Freckle/App/Prelude.hs

```diff
@@ -0,0 +1,137 @@
+-- | Those functions and types we can't do without!
+module Freckle.App.Prelude
+  (
+  -- * 'Prelude' as the starting point, without common partial functions
+    module Prelude
+
+  -- * Commonly imported types
+  , Alternative
+  , Exception
+  , Generic
+  , HashMap
+  , Hashable
+  , Int64
+  , Map
+  , MonadIO
+  , MonadReader
+  , MonadUnliftIO
+  , NominalDiffTime
+  , NonEmpty
+  , PrimMonad
+  , ReaderT
+  , Set
+  , Text
+  , UTCTime
+  , Vector
+
+  -- * Commonly used functions
+
+  -- ** Lifts
+  , lift
+  , liftIO
+
+  -- ** 'Text'
+  , tshow
+
+  -- ** 'Maybe'
+  , catMaybes
+  , fromMaybe
+  , isJust
+  , isNothing
+  , listToMaybe
+  , mapMaybe
+  , maybeToList
+  , readMay
+
+  -- ** 'Either'
+  , partitionEithers
+
+  -- ** 'Foldable'
```

Comment:
Honestly, I'd just export all of `Foldable` and `Traversable`, maybe with the `_1` functions hidden, in favor of their `semigroupoid` version.

Reply:
Sounds good but aren't the `_{n}` functions from `Control.Lens`?

Reply:
Oh sorry, I meant things like `foldl1`, not `_1` and `_2`. Maybe `____1` would have been more understandable?

Reply:
Oh, gotcha!